pages/update/_id.vue

```diff
@@ -27,7 +24,6 @@ export default {
             error({statutCode:500, message:'Impossible de trouver la licorne'})
         }
     },
-
     methods: {
         async editUnicorn(form){
             try{
```

Comment:
ligne 40, pourquoi ne pas utiliser le this.$router.push('/') ?

Reply:
je trouve ça plus intéressant de retourner sur la page de détail quand on a fait une modification pour que l'utilisateur puisse voir ses modifications